x-pack/plugins/security_solution/public/management/pages/trusted_apps/view/components/trusted_app_card/index.tsx

```diff
@@ -120,6 +120,26 @@ export const TrustedAppCard = memo<TrustedAppCardProps>(
             <TextFieldValue fieldName={PROPERTY_TITLES.created_by} value={trustedApp.created_by} />
           }
         />
+        <ItemDetailsPropertySummary
+          name={PROPERTY_TITLES.updated_at}
+          value={
+            <FormattedDate
+              fieldName={PROPERTY_TITLES.updated_at}
+              value={trustedApp.updated_at}
+              className="eui-textTruncate"
+            />
+          }
+        />
+        <ItemDetailsPropertySummary
+          name={PROPERTY_TITLES.updated_by}
+          value={
+            <FormattedDate
+              fieldName={PROPERTY_TITLES.updated_by}
+              value={trustedApp.updated_by}
+              className="eui-textTruncate"
```

Comment:
Not 100% sure if we need to truncate the updated name. The creator name is not truncated but I added it here anyway.